model/event/subscriptions.go

```diff
@@ -34,8 +34,10 @@ var (
 type OwnerType string
 
 const (
-	OwnerTypePerson  OwnerType = "person"
-	OwnerTypeProject OwnerType = "project"
+	OwnerTypePerson      OwnerType = "person"
+	OwnerTypeProject     OwnerType = "project"
+	TaskDurationKey                = "task_duration_secs"
+	TaskPercentChangeKey           = "task_percent_change"
```

Comment:
Since we seem to be preferring hyphens for things like this, should they be hyphens.

Reply:
changed